src/client/results/Feed.js

```diff
@@ -29,32 +36,78 @@ class Feed extends Component {
           </div>
         </div>
         <div className="ui divided items">
-          {paginatedFeed.map((feedItem) => (<FeedItem content={feedItem} key={feedItem.data.id}/>))}
+          {paginatedFeed.map((feedItem) => (
+            <FeedItem content={feedItem} key={feedItem.data.id} />
+          ))}
         </div>
         <div>
-          <PaginationButtons numberOfPages={Math.ceil(filteredFeed.length / paginationInfo.limit)} paginationInfo={paginationInfo}/>
+          <PaginationButtons
+            numberOfPages={Math.ceil(filteredFeed.length / paginationInfo.limit)}
+            paginationInfo={paginationInfo}
+          />
         </div>
       </div>
     );
   }
-};
-
+}
 Feed.propTypes = {
   feed: React.PropTypes.array,
   paginationInfo: React.PropTypes.object,
-  hiddenWords: React.PropTypes.array,
-  hiddenUsers: React.PropTypes.array,
+  toggledWords: React.PropTypes.array,
+  isWordsToggledActionHide: React.PropTypes.bool,
+  toggledUsers: React.PropTypes.array,
+  isUsersToggledActionHide: React.PropTypes.bool,
+};
+
+const filterPostsForFeed = (
+  feed, toggledWords, isWordsToggledActionHide, toggledUsers, isUsersToggledActionHide
+) => (
+  feed.filter((feedItem) => {
+    const content = feedItem.data.content;
+    const authorId = feedItem.author.id;
+
+    // If we can find the chosen hidden word in this tweet, block the post
+    for (const toggledWord of toggledWords) {
+      if (content.indexOf(`${toggledWord}`) > -1) {
```

Comment:
Any reason this isn't just `toggledWord` directly rather than wrapping in backticks?